sentry-core/Cargo.toml

```diff
@@ -16,23 +16,19 @@ edition = "2018"
 all-features = true
 
 [features]
-default = ["with_client_implementation"]
-with_client_implementation = ["im", "url", "rand"]
-with_debug_to_log = ["log"]
-with_test_support = []
+default = []
+client = ["im", "rand"]
+debug-to-log = ["log"]
```

Comment:
_nit_: Maybe even shorter `debug-logs`? :)

I think we could remove this feature and just use the `log` feature from the optional dependency. The only reason not to do that would be in the top-level `sentry` crate, if we want to distinguish between `log` support and the internal debug logging. 

In the rest of the code base, it seems that you're mostly using the `log` feature for this already.

Reply:
This is a bit of a 🤦‍♂️ moment. cfg is not really hygienic in macros, which means those features need to have the same name across core and sentry.

Reply:
I see, in that case, probably having one consistent feature with a name distinct from `log` would be better.